platform/docs/docs/configuration/url.md

```diff
@@ -110,17 +110,31 @@ You can open more than one study in the Viewer by adding the `StudyInstanceUIDs`
 :::tip
 
 You can ues this feature to open a current and prior study in the Viewer.
-Read more in the [Hanging Protocol Module](../platform/extensions/modules/hpModule.md#matching-on-prior-study-with-uid) section
+Read more in the [Hanging Protocol Module](../platform/extensions/modules/hpModule.md#matching-on-prior-study-with-uid) section.  You can also use commas to separate
+values.
 
 :::
 
 
-### SeriesInstanceUIDs
+### SeriesInstanceUID and displaySet.SeriesInstanceUID
 
-Sometimes you need to only open a specific series in a study, you can do that by
+Sometimes you need to only retrieve a specific series in a study, you can do that by
 
 ```js
 /viewer?StudyInstanceUIDs=1.3.6.1.4.1.25403.345050719074.3824.20170125095722.1&SeriesInstanceUID=1.3.6.1.4.1.25403.345050719074.3824.20170125095748.1
 ```
 
 This will only open the viewer with one series (one displaySet).
+
+Alternatively, you can open the study on a specified series and/or sop instance
+using:
+`displaySet.SeriesInstanceUID` and/or `displaySet.SOPInstanceUID`
```

Comment:
as discussed on the call, `displaySet.` might be a little bit weird (need dicom prior knowledge kind of), how about `initialSeriesInstanceUID` and `initialSOPInstanceUID`